CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.2.1
+-----
+**Date:** 2016-11-28
+
+Fix potential Deadlock in Checkpoints & QED-RR
```

Comment:
Fix potential deadlock in checkpoints & highlight QED nonlinear Thomson scattering